src/Api/ApiCalls.js

```diff
@@ -1,11 +1,36 @@
 import axios from '../config/axios';
 import { isLoggedIn, Login } from '../redux/slices/auth';
 
-export default async function UserLogin(data, dispatch) {
+export default async function UserLogin(data, dispatch, error, setLoading) {
```

Comment:
[Optional] This one looks and works very well, but I would recommend following the **S**OLID Single Responsibility Principle. For example in here we're passing multiple arguments, but we can achieve this by returning a promise, and in the `then` method of our promise we can manage the next steps. It'll help us to avoid passing extra pieces of stuff to our method.